### ISSUE TITLE
Make a test case for trying to decrypt an empty file (and random files of various lengths).

### ISSUE THREAD

**Issue by defuse** [defuse/php-encryption#131]
```
[firexware@firexware-pc php-encryption]$ cat test.php 
<?php

require 'autoload.php';

echo "Creating a key...\n";
$key = \Defuse\Crypto\File::createNewRandomKey();

echo "Decrypting /tmp/empty.txt...\n";
\Defuse\Crypto\File::decryptFile("/tmp/empty.txt", "/tmp/empty.txt.dec", $key);
echo "Done.\n";
[firexware@firexware-pc php-encryption]$ cat /tmp/empty.txt
[firexware@firexware-pc php-encryption]$ php test.php 
Creating a key...
Decrypting /tmp/empty.txt...
^C
```

**Comment by defuse** [comment]
(Make sure there's a test case for this.)

**Comment by defuse** [comment]
Pretty sure this is fixed now. Just need to make sure there's a test case.

**Comment by defuse** [comment]
This is included in #148.

**Comment by defuse** [comment]
Done.
